Formula/creduce.rb

```diff
@@ -15,6 +15,7 @@ class Creduce < Formula
   depends_on "astyle"
   depends_on "delta"
   depends_on "llvm"
+  depends_on "clang-format"
```

Comment:
This should go above the `delta` dependency; standard deps are sorted alphabetically, or at least should be. Should also bump the `revision` of the formula if you're changing dependencies.